test/topology/util.py

```diff
@@ -234,6 +236,9 @@ async def check_system_topology_and_cdc_generations_v3_consistency(manager: Mana
         for row in topo_res:
             logging.info(f"  {row}")
 
+    # Filter out ignored nodes
+    topo_results = [[row for row in topo_res if row.host_id not in ignored_nodes] for topo_res in topo_results]
+
```

Comment:
A similar problem was recently fixed in scylla-dtest: scylladb/scylla-dtest@3243de54a8c07da60cea9417cc7d5ea918b1e095

but a different approach was taken: checking that nodes expected to be normal are normal, and nodes expected to be left are left.

You could take that approach here for consistency if it's not too painful.

But I guess it's a minor thing, this approach works too.

Reply:
Updated accordingly.

By the way, I think there is a small issue in the dtest, see the comment that I left there.

Reply:
> see the comment that I left there

where exactly?

Reply:
Under the PR that the commit originated from (yeah, not obvious, required two clicks instead of one :P): https://github.com/scylladb/scylla-dtest/pull/5429#discussion_r1930183849